utils/pixiv.py

```diff
@@ -103,6 +104,12 @@ def recompile_date(date: str):
             "%Y년 %m월 %d일"
         )
 
+    @staticmethod
+    def html2text(html: str):
+        soup = BeautifulSoup(html, 'html.parser')
+        text_parts = soup.findAll(text=True)
+        return ''.join(text_parts)
+
```

Comment:
cant use .replace("<br>","\n")?

Reply:
In case of `87459857`, the comment section is as follows.
```
Twitter→ <strong><a href="https://twitter.com/HAAAM08" target="_blank">twitter/HAAAM08</a></strong>
```
Therefore, completely removed the html tags and converted them to text.

Reply:
@ombe1229 
A dependency change is required.

Add that dependency in requirements.txt